lib/installHelpers.js

```diff
@@ -578,3 +579,71 @@ function error(msg) {
 function isSilent() {
   return process.env.SILENT;
 }
+
+/**
+* Manually collect all misisng prerequisites
+* Therefore we do not pass error to async-callbacks as this would exit the process
+*/
+function checkPrerequisites(callback) {
+  const result = {
+    errors: [],
+    warnings: []
+  };
+
+  async.applyEach([
+    checkGitInstalled,
+    checkGruntCliInstalled,
+    checkGithubConnection,
+    checkLocalMongoDbConnection
+  ], result, function(error) {
+    if (error) return callback(error);
+    if (result.errors.length) {
+      return callback(new Error(result.errors.join(', ')));
```

Comment:
Suggest a line break between errors.

Reply:
have tried this, will format it oddly. I can do however if you like.

Reply:
This is the output for me in Windows if changed to `.join('\n')`:

![](https://user-images.githubusercontent.com/922987/44102675-30f7d3ce-9fe2-11e8-8de2-2caa1d8d21c5.png)

Reply:
Looks good.

Agree that it would look nicer with each error on a separate line.
```
Missing the following pre-requisites, please check install guides:
- git
- grunt
- ffmpeg
- failed to connect to github.com
```